composer.json

```diff
@@ -8,30 +8,26 @@
         "cweagans/composer-patches": "~1.0",
         "drupal/admin_toolbar": "^3.0",
         "drupal/autocomplete_id": "^1.4",
-        "drupal/backup_migrate": "^4.0",
         "drupal/cer": "^4.0@alpha",
-        "drupal/core-composer-scaffold": "^8.9",
-        "drupal/core-project-message": "^8.9",
-        "drupal/core-recommended": "^8.9",
-        "drupal/core-vendor-hardening": "^8.9",
-        "drupal/devel": "^2.1",
+        "drupal/core-composer-scaffold": "^9.0.0",
+        "drupal/core-project-message": "^9.0.0",
+        "drupal/core-recommended": "^9.0.0",
+        "drupal/core-vendor-hardening": "^9.0.0",
         "drupal/diff": "^1.0",
         "drupal/elasticsearch_connector": "^7.0-alpha2",
         "drupal/entity": "^1.2",
         "drupal/field_group": "^3.1",
         "drupal/flood_unblock": "^2.0",
-        "drupal/flysystem": "~1.0",
+        "drupal/flysystem": "^2.0",
         "drupal/flysystem_s3": "^2.0-rc1",
-        "drupal/govuk_design_system": "^1.0@beta",
-        "drupal/govuk_inline_form_errors": "^1.0@beta",
+        "drupal/govuk_design_system": "^2.0",
+        "drupal/govuk_inline_form_errors": "2.0.0-beta2",
         "drupal/health_check": "^1.1",
-        "drupal/jsonapi_cross_bundles": "^1.0@alpha",
+        "drupal/jsonapi_cross_bundles": "dev-3172761-unable-to-install#c0c90e6ecdd894732a0ca58f6d6c314e9dc29f28",
```

Comment:
The support for Drupal 9 for this module has not been fully released, so we have to attach to a commit.